### ISSUE TITLE
Short project feedback

### ISSUE THREAD

**Issue by danielhe4rt** [vbasilioo/pomodoro#1]
## Current Feedback

There's room to improve but so far, I liked the the project at the first sight. I'll drop some questions among other ideas for this refinement. 

### Positive Points

* Really good usage of Seeders (something rare between my reviews)
* Permissions with Filament used in inpressive ways.
* Livewire also fitted well the project. 

### Enhancement Points

* Lack of [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) and regular/minor commits.
    * First thing that I usually do in a repository is to check the latest commits to understand the project development overview and I couldn't do it due: you guys committed the project bootstrap "composer create-project laravel project name" with many other features together;
    * In the upcoming commits, try to split the commits by features with a better description, and please don't do anything besides parenthesis in the name. E.g.:
        * [x] feat(blade/views/faces): eyes toggling and movement
        * [x] < Feat >[Blade] something here
    * PS: the idea is to choose a domain that makes sense with the project, so any of blade/views/faces would fit.
* Decide if will use a package or craft a system by yourselves. The `spatie` package do the work for you guys, so there's no need for a new model `UserRole` or `UserPermission`. Just use the methods that they provide and add some docs on top of it.
* Migrations should be splitted in different files or it can't be considered properly `Migrations`. This attitude can be harmful in a need of a `php artisan migrate:rollback` by erasing the whole database.
    * For now, keep it since it will be a pain in the ass to revert. 
* Weak typing into functions. Please, consider static analysis tools such as: Larastan and Pint; 

### Support Material

* https://www.conventionalcommits.org/en/v1.0.0/
* https://github.com/larastan/larastan
* https://laravel.com/docs/11.x/pint
* https://laravel.com/docs/11.x/eloquent-relationships#defining-custom-intermediate-table-models

**Comment by vbasilioo** [comment]
Hey bro @danielhe4rt, thanks for the feedback.

I attended to everyone and adjusted:

1. Now all migrations are separated into their respective files to better serve artisan:rollback
2. I implemented Laravel Larastan and Laravel Pint, these are libraries that I have been looking for for a long time; There are some errors still presented but they are pointed out to Liveware and to some stronger typings that PHP does not support, would you know how to resolve them?
3. I'm starting to implement the commit pattern presented, I don't know if it's still in the best shape, but I'll continue breaking down the code commits.
4. Regarding the permissions and roles models, it was really a mistake on my part that I forgot to import Spatie's own model for permissions and the like, now it is corrected and both have been removed from the code's native models.

Taking this opportunity to ask, do you know of any library or way to implement the UUID in a way that Filament accepts at Login? Although it appears OK in the database, when logging in, Filament "does not accept", logging in but not redirecting the user to the administrative panel.

**Comment by danielhe4rt** [comment]
I don't think that there's any specific library. You should simply use `HasUuid` and it would work.

Remember that the products are the Faces itself, so make it purchaseable at some point.
